deployment/pypi/install.ps1

```diff
@@ -0,0 +1,48 @@
+param([bool]$version_ts=$false)
```

Comment:
I install it successfully with conda environment, but I get this error message "ERROR: Fail to find nni under python library".

Reply:
Maybe the old nni is being used by another process, and install new nni can not install it into the right place. You can check the new nni install log. Thanks.

Reply:
tar problem is fixed, please check

Reply:
Thank you.

Reply:
It seems that I still cannot get it work with my anaconda environment. (PS: I installed nni in wsl before and I uninstall it)

Reply:
Does it show the same error? Fail to find nni under python library? Can you send me the cmd stdout content. Thanks.

Reply:
After I added `[Net.ServicePointManager]::SecurityProtocol = "tls12, tls11, tls"` in the top of install.ps1 , I build the nni and run the mnist-keras example successfully in windows mode.